testnet1/genesis.json

```diff
@@ -188,7 +197,14 @@
         "ethereum_address": "0xa8CE9eF6ac6C4a1cE18aEd13bA1EFfa718c0e481",
         "info_url": "https://lovali.xyz",
         "country": "FI"
-      }
+      },
+      "Pk0RvTDEjG8h4vH0s2Zk5+TZLAgdJ8HETaovD++76hg=": {
+        "vega_pub_key": "df94148a3c7b8b6d07e4e69ff95ca011405cf0078659decd0d5aab0232b2e4ef",
+        "ethereum_address": "0x4E34DdF693EcCcc184cbc62A825C96942a7F2676",
+        "tm_pub_key": "",
+        "info_url": "https:/rbf.capital/staking",
```

Comment:
Can you fix the URL it looks broken (missing forward slash after HTTPS)

Reply:
Done